### PR TITLE
Add clarification for readme regarding Espruino

### PR DESCRIPTION
**Pull request by aidancuckoo**
Add clarification for Espruino as I tried using Espruino on my v1.5 micro:bit but Espruino doesn't work with the LSM303AGR on the v1.5 micro:bit due to different I2C addresses. I [added an issue](https://github.com/espruino/EspruinoDocs/issues/537) however @gfwilliams hasn't fixed this due to low interest from the entire micro:bit community - I'll try and make a PR for the Micro:bit Educational Foundation's website if it's hosted on GitHub. Thanks! 

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

- [ x]  I have read and understood the [Contribution Guidelines](htps://github.com/carlosperate/awesome-microbit/blob/master/contributing.md)
- [x] I am making an individual pull request for each suggestion
- [x] I have used the correct spelling and capitalisation for `BBC micro:bit`, `micro:bit`, and `Micro:bit Educational Foundation` (resource title could be excluded if there is a good reason)
- [x] The content linked is in English, or it contains an English translation
- [x] I have added the new entry to the bottom of its the category
- [x] I have used the following format:
```
- [Resource Title](link) - Resource short Description (2 lines or less in total)
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ Contributions are welcome! Not sure how to submit a contribution? Have a look at
 - [MakeCode](https://makecode.microbit.org) - This block and text editor for the micro:bit provides an in-browser emulator, a Blocks interface, and JavaScript (TypeScript) editor.
 	- [MakeCode Beta](https://makecode.microbit.org/beta) - Beta version of the MakeCode editor to test the latest features.
 	- [MakeCode Windows 10 App](https://www.microsoft.com/store/apps/9pjc7sv48lcx) - Windows 10 application for micro:bit MakeCode.
-- [Espruino JavaScript](https://www.espruino.com/MicroBit) - JavaScript interpreter for microcontrollers. It also offers a WebIDE for written code and blocks.
+- [Espruino JavaScript](https://www.espruino.com/MicroBit) - JavaScript interpreter for microcontrollers. It also offers a WebIDE for written code and blocks. Check your micro:bit if it has a combined sensor or two individual sensors - the former doesn't work.
```
Comment (carlosperate) [inline README.md]:
```suggestion
- [Espruino JavaScript](https://www.espruino.com/MicroBit) - JavaScript interpreter for microcontrollers. It also offers a WebIDE for written code and blocks. It's currently not compatible with v1.5 of the micro:bit (due to the different motion sensors).
```

**Comment by aidancuckoo** [comment]
Okay - done!

**Comment by carlosperate** [comment]
Great, thanks @aidancuckoo!

**Comment by gfwilliams** [comment]
Just a small comment - the microbit build still works great on the other board for everything apart from the compass/accelerometer so the wording of this might be slightly misleading.

**Comment by carlosperate** [comment]
> the microbit build still works great on the other board for everything apart from the compass/accelerometer so the wording of this might be slightly misleading.

That's a fair point, are they accelerometer and magnetometer objects in Espruino? Would this be a better description?

```markdown
JavaScript interpreter for microcontrollers. It also offers a WebIDE for written code and blocks. The motion sensors from v1.5 of the micro:bit are currently not implemented.
```

**Comment by aidancuckoo** [comment]
I'll add this for now...

**Comment by gfwilliams** [comment]
Thanks!

> are there accelerometer and magnetometer objects in Espruino?

There are two global functions: https://www.espruino.com/MicroBit#-acceleration-

If you don't use those then you'll be fine - everything else will work *including* the full Bluetooth LE stack. 

I'm not against adding support for the new sensors at some point either if the port really is being used (or accepting a PR!) but I have received virtually nothing in the way of feedback - the first mention of the micro:bit port I had in a year or so was [@aidancuckoo's issue](https://github.com/espruino/EspruinoDocs/issues/537). Unfortunately I'm flat out with work at the moment so until I know people are actually going to use this it's quite low priority.

**Comment by carlosperate** [comment]
I see, thanks for the clarification @gfwilliams.
Are you happy with the new wording then? https://github.com/carlosperate/awesome-microbit/pull/333/files

**Comment by gfwilliams** [comment]
That sounds great - thank you!

**Comment by gfwilliams** [comment]
Just updated - there is now 1.5 support: https://github.com/carlosperate/awesome-microbit/pull/364

**Comment by carlosperate** [comment]
That's awesome, thanks Gordon!

**Pull request merged**